assets/js/components/adminbar/AdminBarWidgets.test.js

```diff
@@ -108,4 +111,42 @@ describe( 'AdminBarWidgets', () => {
 			queryByText( /Set up Google Analytics/ )
 		).not.toBeInTheDocument();
 	} );
+
+	it( 'should render the Admin Bar Widgets for the view only user if the module is shared', async () => {
+		provideUserAuthentication( registry, { authenticated: false } );
```

Comment:
Maybe you can include the test case for when both modules (`analytics-4` and `search-console`) are shared as well?

Reply:
IMHO, it's unnecessary. I think verifying the Search Console widgets is enough. Additionally, we have stories to verify the Analytics widgets :)